src/js/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -1,15 +1,24 @@
 import { connect } from 'react-redux';
 import React from 'react';
+import moment from 'moment';
 
 import AlertBox from '../../common/components/AlertBox';
 import CarefulConsiderationStatement from '../components/CarefulConsiderationStatement';
-import { branchOfService, board, formNumber, venueAddress } from '../utils';
+import { branchOfService, board, formData, venueAddress } from '../utils';
 import { venueWarning } from '../config';
 
 class GuidancePage extends React.Component {
+  componentDidMount() {
+    const script = document.createElement('script');
+    script.type = 'text/javascript';
+    script.async = true;
+    script.src = 'https://standards.usa.gov/assets/js/vendor/uswds.min.js';
+    document.getElementsByTagName('head')[0].appendChild(script);
```

Comment:
Why does this page specifically need this?

Reply:
@U-DON the USWDS accordion requires this script :(

There may be a better way to do this but haven't explored it too much in depth

Reply:
Huh. Then wouldn't tons of other pages that use the accordion pattern need this? Why isn't this just part of our includes?

Reply:
Maybe - I was planning to explore further when I had time. The other accordions are all on static pages and people drop the script tag into the bottom of markdown file. Seems like uswds does something onload to initialize the accordions so I am getting it to work here by injecting it